File: src/main/java/model/Game.java
package model;

import java.io.Serializable;

public class Game extends Model implements Serializable {
    @SuppressWarnings("WeakerAccess")
    public String name;

    @SuppressWarnings("WeakerAccess")
    public String link;

    @SuppressWarnings("WeakerAccess")
    public String image;

    public Game(String name, String link, String image) {
        this.name = name;
        this.link = link;
        this.image = image;
    }
}

File: src/main/kotlin/model/Game.kt
package model

import java.io.Serializable

data class Game(
        var name: String,
        var link: String,
        var image: String
) : Model(), Serializable {
    override fun save() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}
